example/unit-testing/complex/tests/src/test_protocol.cpp

```diff
@@ -54,20 +54,19 @@ TEST(TestProtocolParser, Hello) {
   };
 
   mock()
-      .expectOneCall("a")
```

Comment:
Not sure if this was an intentional failure for demonstration purposes, but I think it's better to let readers introduce their own failures.